### PR TITLE
Ignore aspectMask if attachment is unused

### PR DESCRIPTION
**Pull request by krOoze**

### CODE REVIEW

File: chapters/renderpass.txt [hunk 1]
```diff
@@ -1871,15 +1871,14 @@ ifdef::VK_NVX_multiview_per_view_attributes[]
...
-    must: be a valid combination of elink:VkImageAspectFlagBits
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-03176]]
-    The pname:aspectMask member of any element of pname:pInputAttachments
-    must: not be `0`
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-02529]]
-    The pname:aspectMask member of each element of pname:pInputAttachments
-    must: not include ename:VK_IMAGE_ASPECT_METADATA_BIT
+  * The pname:aspectMask member of any element of pname:pInputAttachments
+    that is not ename:VK_ATTACHMENT_UNUSED must: be a valid combination of
+    elink:VkImageAspectFlagBits
```
[the remark this answers is not in the document]
Comment (krOoze) [reply chapters/renderpass.txt]: very well

File: chapters/renderpass.txt [hunk 2]
```diff
@@ -1871,15 +1871,15 @@ ifdef::VK_NVX_multiview_per_view_attributes[]
...
-    The pname:aspectMask member of each element of pname:pInputAttachments
-    must: not include ename:VK_IMAGE_ASPECT_METADATA_BIT
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: be a
+    valid combination of elink:VkImageAspectFlagBits
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: not
+    be `0`
+  * If pname:attachment member of any element of pname:pInputAttachments is
+    not ename:VK_ATTACHMENT_UNUSED, then pname:aspectMask member must: not
```
Comment (Tobski) [inline chapters/renderpass.txt]: grammatical nit: should say something like "then _its_ pname:aspectMask member" or "then _that element's_ pname:aspectMask member"

Comment (Tobski) [reply chapters/renderpass.txt]: (Same for all three)

File: chapters/renderpass.txt [hunk 3]
```diff
@@ -1871,15 +1871,15 @@ ifdef::VK_NVX_multiview_per_view_attributes[]
...
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-03175]]
-    The pname:aspectMask member of any element of pname:pInputAttachments
-    must: be a valid combination of elink:VkImageAspectFlagBits
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-03176]]
-    The pname:aspectMask member of any element of pname:pInputAttachments
-    must: not be `0`
-  * [[VUID-VkSubpassDescription2KHR-aspectMask-02529]]
-    The pname:aspectMask member of each element of pname:pInputAttachments
-    must: not include ename:VK_IMAGE_ASPECT_METADATA_BIT
+  * If pname:attachment member of any element of pname:pInputAttachments is
```
Comment (oddhack) [inline chapters/renderpass.txt]: If *the* pname:attachment member... (same for the other two VUs).

File: chapters/renderpass.txt (hunk 2 above)
Comment (krOoze) [reply chapters/renderpass.txt]: I kinda hate "its" in specification. Somehow it felt clearer to me without it.
With "it" it is not clear if it refers to member of pname:attachment or member of element of pname:pInputAttachments. With it ommited it feels like we talk about members of the same entity in both instances in the sentence.

Comment (Tobski) [reply chapters/renderpass.txt]: "the" is fine by me.

**Pull request merged**